RemotingSample/Assets/Materials/Black0Alpha.mat

```diff
@@ -0,0 +1,78 @@
+%YAML 1.1
```

Comment:
I don't think you're using these Black0Alpha and Black1Alpha materials, right?
These were part of an old rendering test

Reply:
removed